.github/workflows/check-versions.yml

```
@@ -13,7 +13,7 @@ jobs:
       - name: Use Node.js ${{ matrix.node-version }}
         uses: actions/setup-node@v1
```

While we're here, I recommend
```suggestion
        uses: actions/setup-node@v3
```